src/cpp/r/R/Tools.R

```diff
@@ -281,7 +281,7 @@ environment(.rs.Env[[".rs.addFunction"]]) <- .rs.Env
    # dependencies may live within a separate library path from the package to be
    # loaded; it should suffice to place the requested library at the front of
    # the library paths
-   if (nzchar(lib)) {
+   if (nzchar(lib) && file.exists(file.path(lib, packageName, "Meta"))) {
```

Comment:
Or should it instead actually not load the package at all in that case ?

Reply:
Maybe that should still stay on, just in case ? `library()` won't be able to load the package if it's not an installed package